tests/cross/src/test/scala/tests/pc/SignatureHelpDocSuite.scala

```diff
@@ -125,6 +131,16 @@ class SignatureHelpDocSuite extends BaseSignatureHelpSuite {
             |        ^^^^^^^^^^^^^
             |  @param ifEmpty String the expression to evaluate if empty.
             |  @param f the function to apply if nonempty.
+            |""".stripMargin,
+      "3" ->
+        s"""|$foldLatestDocs
+            |**Parameters**
+            |- `f`: the function to apply if nonempty.
+            |- `ifEmpty`: the expression to evaluate if empty.
+            |fold[B](ifEmpty: => B)(f: A => B): B
```

Comment:
It seems that we need  `seenFrom` as in [HoverProvider](https://github.com/scalameta/metals/blob/806786093d6f3cf8f9e4aa490dbe3a94662f1230/mtags/src/main/scala-3/scala/meta/internal/pc/HoverProvider.scala#L140) but we can do it later

Reply:
I will do an update with that!

Reply:
Follow up here: https://github.com/scalameta/metals/pull/3941